source/nodejs/adaptivecards-designer/src/adaptivecards-designer.ts

```diff
@@ -10,9 +11,18 @@ export const defaultMicrosoftHosts: containers.HostContainer[] = [
 	new containers.OutlookContainer("Outlook Actionable Messages", "containers/outlook-container.css"),
 	new containers.DarkTeamsContainer("Microsoft Teams - Dark", "containers/teams-container-dark.css"),
 	new containers.LightTeamsContainer("Microsoft Teams - Light", "containers/teams-container-light.css"),
-	new containers.VivaConnectionsContainer("Viva Connections (Preview)", "containers/viva-connections-container.css"),
+	new containers.VivaConnectionsContainer("Viva Connections", "containers/viva-connections-container.css"),
 	new containers.DarkCortanaContainer("Cortana Skills - Dark", "containers/cortana-container-dark.css"),
 	new containers.LightCortanaContainer("Cortana Skills - Light", "containers/cortana-container-light.css"),
 	new containers.BotFrameworkContainer("Bot Framework Other Channels (Image render)", "containers/bf-image-container.css"),
 	new containers.CortanaClassicContainer("Cortana Skills (Classic)", "containers/cortana-classic-container.css"),
-]
+	new containers.FederatedSearchContainer("Federated Search","containers/federated-search-container.css"),
```

Comment:
Do you expect this change?

Reply:
Yes the federated search host app is included in this release.